app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  before_action :current_user, only: [:create]
+
+  def create
+    @comment = current_user.comments.new(comments_params)
+    @comment.users_id = current_user.id
+    @comment.post_id = params[:post_id]
```

Comment:
- [ ] <b>REQUIRED:</b> When you say `comment = current_user.comments.new()` the user_id is already initialize, so you don't need to re-initialize it again as you did on line 6. meaning your code would simply be like this:
```rb
@comment = current_user.comments.new(params)
@comment.post_id = params[:post_id]
```